### DISCUSSION TITLE
Shimmer beta und die normale

### DISCUSSION THREAD

**Discussion by Ninjakek** [iotaledger/firefly#4842] [Help & Support]
ich bins nochmal, bei mir ging das claimen auf der beta das problem ist nur jetzt hab ich zwei shimmer firefly wallets die eine ist 2.0.0 beta die andere ist die normale 2.0.3 , doch ich hab auf der beta geclaimd und ich kann sie nirgends hinsenden , also werde ich warten müssen bis irgendwann die shimmer in die Firefly integriert wird. ist mir eigentlich eh scheiß egal mich nervt nur das die einfach das komma verschoben haben .

**Comment by MiKeF05** [comment]
Solange du mit der Beta nicht mit dem Mainnet verbunden warst, wurden deine Token auch nicht geclaimed. Denn bei den Adressen rms.. handelt es sich um Token im Testnet, die keinerlei Wert haben. 

Das mit der Kommaverschiebung muss dich nicht nerven, weil sich der Wert deiner Token nicht verändert hat. Es hat sich lediglich die Einführung von 6 Kommastellen geändert.
